steps/setup_docker_continue_generic.yml

```diff
@@ -8,6 +8,14 @@ hints:
 - Make sure you ran the \`docker-compose up\` command from the **project root directory**
   and that it finished running. This may take a few moments.
 startFlow:
+  if:
+    conditions:
+    - conditionId: user_has_feature_flag
```

Comment:
I do not understand the condition format.
in some cases is like here, and in others, there is "equal" property...

Reply:
yeah it's not intuitive. without the `equals` property is the same as `equals: true` property, it's a shorthand. cc @esapir

Reply:
OK got it

Reply:
Yep, default for equals is true. It does not have to be a boolean (although for now it always is)